ext/random/tests/03_randomizer/construct_twice.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+Random: Randomizer: __construct() twice
+--FILE--
+<?php
+try {
+    (new \Random\Randomizer())->__construct();
+} catch (\BadMethodCallException $e) {
+    echo $e->getMessage() . PHP_EOL;
+}
+
+try {
+    (new \Random\Randomizer(
+        new class () implements \Random\Engine {
+            public function generate(): string
+            {
+                return \random_bytes(4); /* 32-bit */
+            }
+         }
```

Comment:
I believe the test would be more readable, if this was an actual named class instead of an anonymous class. I would also use a variable for the randomizer to split this into two lines:

```
$r = new \Random\Randomizer(new UserEngine());
$r->__construct(new UserEngine());
```

The variable also applies to the test case above.

Reply:
fixed: cb5f7aeb8040e04807ce975a42497a74b84f244c